packages/sentry_crispin_transport/pubspec.yaml

```diff
@@ -0,0 +1,17 @@
+name: sentry_crispin_transport
+description: Sentry Logger for Crispin
+version: 1.0.0+1
+publish_to: none
+
+environment:
+  sdk: ">=2.17.1 <3.0.0"
+
+dev_dependencies:
+  coverage: ^1.1.0
+  mocktail: ^0.2.0
+  test: ^1.19.2
+  very_good_analysis: ^2.4.0
+dependencies: 
+  crispin: 
+    path: ../crispin
```

Comment:
@Rohithgilla12 does this work in publishing? If so amazing I didn't realize you could do that

Reply:
Interesting question 
Actually it won't work
Good catch, sorry

References

https://github.com/felangel/bloc/blob/0b99ae0c8a156ca3d6f926cdff577a6ee7ff0768/packages/flutter_bloc/pubspec.yaml#L15

https://github.com/firebase/flutterfire/blob/c2771a425bd7260b11970e9e9e77ef40a39f9f16/packages/firebase_auth/firebase_auth/pubspec.yaml#L19

Will update them and add docs 😄